tests/mock_tables/asic1/asic_db.json

```diff
@@ -1,11 +1,11 @@
 {
   "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY:{\"bridge_id\":\"oid:0x0\",\"bridge_type\":\"SAI_FDB_ENTRY_BRIDGE_TYPE_1Q\",\"mac\":\"7C:FE:90:80:9F:10\",\"switch_id\":\"oid:0x21000000000000\",\"vlan\":\"1000\"}": {
-    "SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID": "oid:0x3a0000000006208",
+    "SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID": "oid:0x3a000000000616",
```

Comment:
what trigger all these below changes ?

Reply:
The changes in mock tables are done so that the SAI ID for port and other objects are same in different namespace. Earlier, in the mock tables, it was unique across the device. To reflect the actual scenario where the same SAI Id can appear in a different namespace, the changes are made in asic_db and counter_db.